<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(52, 52, 52);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>1061</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>                                    HEY BUDDY ,WELCOME TO MHT CET COLLEGE PREDICTOR FOR 2022 !!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Recommended_Button">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>591</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 0, 0);
border-radius:40px</string>
    </property>
    <property name="text">
     <string>GET RECOMMENDATIONS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Feedback_Button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>591</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: rgb(85, 255, 255);

border-radius:40px</string>
    </property>
    <property name="text">
     <string>   GIVE FEEDBACK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Rank_Button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>610</y>
      <width>591</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 0, 0);
border-radius:40px</string>
    </property>
    <property name="text">
     <string>GET RANK</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
